include/librealsense2/h/rs_option.h

```diff
@@ -122,6 +129,28 @@ extern "C" {
     } rs2_rs400_visual_preset;
     const char* rs2_rs400_visual_preset_to_string(rs2_rs400_visual_preset preset);
 
+    /** \brief For L500 devices: provides optimized settings (presets) for specific types of usage. */
+    typedef enum rs2_l500_visual_preset
+    {
+        RS2_L500_VISUAL_PRESET_CUSTOM,
+        RS2_L500_VISUAL_PRESET_DEFAULT,
+        RS2_L500_VISUAL_PRESET_NO_AMBIENT,
+        RS2_L500_VISUAL_PRESET_LOW_AMBIENT,
+        RS2_L500_VISUAL_PRESET_MAX_RANGE,
+        RS2_L500_VISUAL_PRESET_SHORT_RANGE,
+        RS2_L500_VISUAL_PRESET_COUNT
+    }rs2_l500_visual_preset;
+    const char* rs2_l500_visual_preset_to_string(rs2_l500_visual_preset preset);
+
+    /** \brief For setting the camera_mode option */
+    typedef enum rs2_sensor_mode
```

Comment:
This is the sensor resolution... why "mode"?

Reply:
It was the decision after the previous code review